.github/workflows/formatting.yml

```diff
@@ -61,4 +61,11 @@ jobs:
         working-directory: frontend
 
       - name: run pre-commit
+        if: steps.changes.outputs.frontend == 'true'
+        uses: pre-commit/action@v3.0.0
+
+      - name: run pre-commit without pnpm
+        if: steps.changes.outputs.frontend == 'true'
         uses: pre-commit/action@v3.0.0
+        env:
+          SKIP: pnpm-lint
```

Comment:
because we're not installing the NPM stuff unless the PR contains changes to the frontend code. So that check fails for python-only PR's. It's also unnecessary so it was easiest to disable it.